packages/components/src/components/bal-tabs/bal-tabs.tsx

```diff
@@ -136,6 +137,14 @@ export class Tabs {
     this.debounceChanged()
     this.updateTabs()
 
+    const accordion = (this.accordion = this.el.closest('bal-accordion'))
+
+    if (accordion) {
+      accordion.addEventListener('balChange', () => {
```

Comment:
- [ ] you also need to remove the event listener on the lifecycle disconnect

Reply:
`accordion.addEventListener('balChange', this.accodrionChange)`

`accordion.removeEventListener('balChange', this.accodrionChange)`

Reply:
```
accodrionChange() {
this.moveLine(this.getTargetElement(this.value))
}
```